code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -183,6 +183,9 @@
 	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2, TECH_ESOTERIC = 2)
 	magazine_type = /obj/item/ammo_magazine/pistol/small
 	allowed_magazines = /obj/item/ammo_magazine/pistol/small
+	auto_eject_sound = 'sound/urist/guns/casing_quiet.ogg'
+	mag_insert_sound = 'sound/urist/guns/pistol_magin_quiet.ogg'
+	mag_remove_sound = 'sound/urist/guns/pistol_magout_quiet.ogg'
```

Comment:
This is a non-Urist file, so I would advocate for wrapping this in an `ifdef INCLUDE_URIST_CODE` block (the macro is already supported).

This will make it easier to detect as our change on a merge and (if we applied this consistently) would eventually allow us to compile pure-Bay on Uristcode to check if an issue is theirs or ours.

Reply:
> This is a non-Urist file, so I would advocate for wrapping this in an `ifdef INCLUDE_URIST_CODE` block (the macro is already supported).
> 
> This will make it easier to detect as our change on a merge and (if we applied this consistently) would eventually allow us to compile pure-Bay on Uristcode to check if an issue is theirs or ours.

I'll take a look at doing this so we can avoid some issues in the future, gotcha